src/import-csv.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import Promise from 'bluebird';
+import CouchBulkImporter from 'couchbulkimporter';
+import parse from 'csv-parse';
+import BulkBadger from 'bulkbadger';
+
+export function cli (file, url, ...csvOptions) {
+  if (!file) {
```

Comment:
we can improve the error handling here:

test for:
 - no url set
 - url no http url

for both we already have a comfortable helper: https://github.com/apache/couchdb-nmo/blob/master/src/utils.js#L15